slices/xfonts-utils.yaml

```diff
@@ -0,0 +1,35 @@
+package: xfonts-utils
+
+essential:
+  - xfonts-utils_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libfontenc1_libs
+      - libfreetype6_libs
+      - xfonts-encodings_encodings
+      - xfonts-utils_utils
+      - zlib1g_libs
```

Comment:
I would probably consider putting the `_scripts` slice in here too.

Reply:
Well, for the functioning of the binaries scripts (if you mean `xfonts-utils_scripts`) are not necessary, hence the separate slice.